client/src/components/Accordion.js

```diff
@@ -7,20 +7,30 @@ var StartButton = require ('./startButton.js')
 module.exports = class Accordion extends React.Component {
   constructor(props) {
     super(props)
+
     this.state = {
-      active: false
+      active: false,
+      gameTopic: ''
     }
+
     this.toggle = this.toggle.bind(this)
+    this.handleGameCreateSubmit = this.handleMessageSubmit.bind(this);
```

Comment:
`this.handleGameCreateSubmit = this.handleMessageSubmit.bind(this);` I believe this is a typo